tendrl/commons/manager/__init__.py

```diff
@@ -12,27 +12,32 @@
 class Manager(object):
     def __init__(
             self,
+            message_handler_thread,
             sds_sync_thread,
-            central_store_thread,
```

Comment:
The best practice is to keep the comma at the end for lists, if the last list element allows it. This gets people in the habit of adding a comma at the end of every list element, regardless of whether it's to be the last and prevent syntax errors.